lib/Doctrine/Migrations/Provider/LazySchemaDiffProvider.php

```diff
@@ -49,7 +49,8 @@ public function createFromSchema() : Schema
         return $this->proxyFactory->createProxy(
             Schema::class,
             static function (&$wrappedObject, $proxy, $method, array $parameters, &$initializer) use ($originalSchemaManipulator) {
-                $initializer   = null;
+                $initializer = null;
+
```

Comment:
```
 49 | ERROR | [x] Expected 1 lines after "return", found 0.
    |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
```

Is also causing trouble here

Reply:
Can this not be fixed?

Reply:
I apologize - should have been more verbose on what the issue actually is. 
Identified the following bug/misbehavior while rolling out the new coding-standard - please see: https://github.com/slevomat/coding-standard/issues/635

Would need to fix it upstream to not receive a false positive here as well. The other option would be to exclude this sniff for the time being. When fixed upstream we can reenable the sniff again

Reply:
No worries. Thanks for explaining. Should we disable the rule for now?

Reply:
I am fine with it - however in other repositories in the doctrine project "we" decided to merge it with this change and let the rule stay for now.

Either way is fine for me and I prepare the change 👍

Reply:
I'm fine with that as long as the build is green.